### ISSUE TITLE
Several languages have merge issues during build after pull from transifex

### ISSUE THREAD

**Issue by raveit65** [mate-desktop/mate-user-guide#25]
#### Expected behaviour
no merge issues after `tx pull` -a -f`

#### Actual behaviour
Several languages have merge issues and `tr` language breaks building.
```
Making all in mate-user-guide
make[1]: Verzeichnis „/media/Work/Programme-linux/Mate-Desktop/git-version/github-matedesktop/mate-user-guide-dist/mate-user-guide“ wird betreten
  GEN      eo/eo.mo
  GEN    eo/
  GEN      nn/nn.mo
  GEN    nn/
  GEN      zh_HK/zh_HK.mo
  GEN    zh_HK/
  GEN      be/be.mo
  GEN    be/
  GEN      xh/xh.mo
  GEN    xh/
  GEN      mai/mai.mo
  GEN    mai/
  GEN      zh_TW/zh_TW.mo
  GEN    zh_TW/
  GEN      zh_CN/zh_CN.mo
  GEN    zh_CN/
Warning: Could not merge zh-CN translation for msgid:
<inlinemediaobject> <imageobject> <imagedata format="PNG" fileref="figures/ibeam_pointer.png"/> </imageobject> <textobject> <phrase>I-beam pointer</phrase> </textobject> </inlinemediaobject> I-beam pointer
Warning: Could not merge zh-CN translation for msgid:
<inlinemediaobject> <imageobject> <imagedata format="PNG" fileref="figures/move_pointer.png"/> </imageobject> <textobject> <phrase>Move pointer.</phrase> </textobject> </inlinemediaobject> Move pointer
Warning: Could not merge zh-CN translation for msgid:
<inlinemediaobject> <imageobject> <imagedata format="PNG" fileref="figures/link_pointer.png"/> </imageobject> <textobject> <phrase>Symbolic link pointer.</phrase> </textobject> </inlinemediaobject> Symbolic link pointer
Warning: Could not merge zh-CN translation for msgid:
<inlinemediaobject> <imageobject> <imagedata format="PNG" fileref="figures/ask_pointer.png"/> </imageobject> <textobject> <phrase>Ask pointer.</phrase> </textobject> </inlinemediaobject> Ask pointer
Warning: Could not merge zh-CN translation for msgid:
<inlinemediaobject> <imageobject> <imagedata format="PNG" fileref="figures/move_panel_object_pointer.png"/> </imageobject> <textobject> <phrase>Move panel object pointer.</phrase> </textobject> </inlinemediaobject> Move panel object pointer
Warning: Could not merge zh-CN translation for msgid:
<inlinemediaobject> <imageobject> <imagedata format="PNG" fileref="figures/movewindow_pointer.png"/> </imageobject> <textobject> <phrase>Move window pointer.</phrase> </textobject> </inlinemediaobject> Move window pointer
  GEN      ku_IQ/ku_IQ.mo
  GEN    ku_IQ/
  GEN      mr/mr.mo
  GEN    mr/
  GEN      pa/pa.mo
  GEN    pa/
  GEN      en_CA/en_CA.mo
  GEN    en_CA/
  GEN      sr/sr.mo
  GEN    sr/
  GEN      id/id.mo
  GEN    id/
  GEN      es_CO/es_CO.mo
  GEN    es_CO/
  GEN      is/is.mo
  GEN    is/
  GEN      nl/nl.mo
  GEN    nl/
  GEN      ka/ka.mo
  GEN    ka/
  GEN      ca@valencia/ca@valencia.mo
  GEN    ca@valencia/
  GEN      ru/ru.mo
  GEN    ru/
Warning: Could not merge ru translation for msgid:
<firstname>Carlos</firstname> <surname>Garnacho Parro</surname> <affiliation> <orgname>GNOME Project</orgname> </affiliation> <email>carlosg@gnome.org</email>
Warning: Could not merge ru translation for msgid:
<inlinemediaobject> <imageobject> <imagedata format="PNG" fileref="figures/not_available_pointer.png"/> </imageobject> <textobject> <phrase>Not available pointer.</phrase> </textobject> </inlinemediaobject> Not available pointer
Warning: Could not merge ru translation for msgid:
The <link linkend="overview-desktop">desktop</link> background is the image or color that is applied to your desktop. You can open <guilabel>Background</guilabel> tabbed section in the <application>Appearance</application> preference tool by right-clicking on the desktop and choosing <guimenuitem>Change Desktop Background</guimenuitem>, as well as from the <menuchoice><guimenu>System</guimenu><guisubmenu>Preferences</guisubmenu></menuchoice> menu.
Warning: Could not merge ru translation for msgid:
<guilabel>Zoom</guilabel>: Enlarges the smaller dimension of the image until it meets the screen edges; the image may be cropped in the other dimension.
Warning: Could not merge ru translation for msgid:
<primary>symbolic link</primary> <secondary>to file or folder, creating</secondary>
Warning: Could not merge ru translation for msgid:
<link linkend="caja"><application>Caja File Manager</application></link> displays your folders and their contents. Use this to copy, move and classify your files, and to access CDs, USB flash drives, and any other removable media. When you choose an item from the <link linkend="places-menu"><guimenu>Places</guimenu> menu</link>, a <application>Caja File Manager</application> window opens showing that location.
Warning: Could not merge ru translation for msgid:
Examples of commands and locations that you can use in the <guilabel>Launcher Properties</guilabel> dialog can be found below.
  GEN      kk/kk.mo
  GEN    kk/
  GEN      ms/ms.mo
  GEN    ms/
  GEN      cmn/cmn.mo
  GEN    cmn/
  GEN      ga/ga.mo
  GEN    ga/
  GEN      fa/fa.mo
  GEN    fa/
  GEN      ast/ast.mo
  GEN    ast/
  GEN      gu/gu.mo
  GEN    gu/
  GEN      ml/ml.mo
  GEN    ml/
  GEN      ca/ca.mo
  GEN    ca/
Warning: Could not merge ca translation for msgid:
When you display the contents of a folder in icon view, you can specify how to arrange the items in the folder. To specify how to arrange items in icon view, choose <menuchoice><guimenu>View</guimenu><guisubmenu>Arrange Items</guisubmenu></menuchoice>. The <guisubmenu>Arrange Items</guisubmenu> submenu contains the following sections:
  GEN      af/af.mo
  GEN    af/
  GEN      ne/ne.mo
  GEN    ne/
  GEN      ta/ta.mo
  GEN    ta/
  GEN      el/el.mo
  GEN    el/
  GEN      hi/hi.mo
  GEN    hi/
  GEN      fi/fi.mo
  GEN    fi/
  GEN      kn/kn.mo
  GEN    kn/
  GEN      mg/mg.mo
  GEN    mg/
  GEN      es/es.mo
  GEN    es/
Warning: Could not merge es translation for msgid:
<guilabel>To move a window, press-and-hold this key then grab the window</guilabel>
  GEN      nb/nb.mo
  GEN    nb/
  GEN      bn/bn.mo
  GEN    bn/
  GEN      uz/uz.mo
  GEN    uz/
  GEN      eu/eu.mo
  GEN    eu/
  GEN      sr@latin/sr@latin.mo
  GEN    sr@latin/
  GEN      vi/vi.mo
  GEN    vi/
  GEN      hr/hr.mo
  GEN    hr/
  GEN      mk/mk.mo
  GEN    mk/
  GEN      te/te.mo
  GEN    te/
  GEN      or/or.mo
  GEN    or/
  GEN      cs/cs.mo
  GEN    cs/
Warning: Could not merge cs translation for msgid:
Use the <application>Keyboard Shortcuts</application> preference tool to customize the default keyboard shortcuts to your requirements.
Warning: Could not merge cs translation for msgid:
If you need to retrieve a file from <guilabel>Trash</guilabel>, you can display <guilabel>Trash</guilabel> and move the file out of <guilabel>Trash</guilabel>. When you empty <guilabel>Trash</guilabel>, you delete the contents of <guilabel>Trash</guilabel> permanently.
Warning: Could not merge cs translation for msgid:
The files and folders you put on the desktop are stored in a special folder within your Home Folder, called <filename>Desktop</filename>. Like any other folder, you can put files and other folders directly into it. The only difference is that anything placed into the Desktop folder will then show up on desktop itself.
Warning: Could not merge cs translation for msgid:
You can also choose Resize from the Window Menu, or press <keycombo> <keycap>Alt</keycap><keycap>F8</keycap></keycombo>. The resize pointer appears. Move the mouse in the direction of the edge you want to resize, or press one of the keyboard arrows keys. The pointer changes to indicate the chosen edge. Now you can use the mouse or the arrow keys to move this edge of the window. Click the mouse or press <keycap>Return</keycap> to accept the change. Press <keycap>Escape</keycap> to cancel the resize action and return the window to its original size and shape.
  GEN      fr/fr.mo
  GEN    fr/
  GEN      sl/sl.mo
  GEN    sl/
  GEN      lt/lt.mo
  GEN    lt/
  GEN      am/am.mo
  GEN    am/
  GEN      bs/bs.mo
  GEN    bs/
  GEN      tr/tr.mo
  GEN    tr/
Warning: Could not merge tr translation for msgid:
<application>Preferred Applications</application> can be found by going to <menuchoice><guimenu>System</guimenu><guisubmenu>Preferences</guisubmenu><guimenuitem>Preferred Applications</guimenuitem></menuchoice>.
Warning: Could not merge tr translation for msgid:
Enter the location of the theme archive file in the location entry. Alternatively, select the theme archive file in the file list. When you have selected the file, click <guibutton>Open</guibutton>.
Warning: Could not merge tr translation for msgid:
<guilabel>Alt/Win key behavior</guilabel>
Warning: Could not merge tr translation for msgid:
The Compose key allows you to combine two keypresses to make a single character. This is used to create an accented character that might not be on your keyboard layout. For example, press the Compose key, then <keycap>'</keycap>, then <keycap>e</keycap> to obtain e-acute character.
Warning: Could not merge tr translation for msgid:
This section describes each of the preferences you can set. For a more task-oriented description of keyboard accessibility, see the <citetitle>MATE Desktop Accessibility Guide</citetitle>.
Warning: Could not merge tr translation for msgid:
<guilabel>Ignore fast duplicate keypresses</guilabel>
Warning: Could not merge tr translation for msgid:
<application>Caja</application> is always running while you are using MATE. To open a new <application>Caja</application> window, double-click on an appropriate icon on the desktop such as <guimenuitem>Home</guimenuitem> or <guimenuitem>Computer</guimenuitem>, or choose an item from <link linkend="places-menu"><guimenuitem>Places</guimenuitem> menu</link> on the top panel.
Warning: Could not merge tr translation for msgid:
From the top panel menubar, choose <menuchoice><guimenu>Places</guimenu><guimenuitem>Home Folder</guimenuitem></menuchoice>.
Warning: Could not merge tr translation for msgid:
Use the <guilabel>Tree</guilabel> in the side pane. For more information, see <xref linkend="goscaja-27"/>.
Warning: Could not merge tr translation for msgid:
The <application>Caja</application> file manager includes an easy and simple to use way search for your files and folders. To begin a search press <keycombo><keycap>Ctrl</keycap><keycap>F</keycap></keycombo> or select the <guibutton>Search</guibutton> toolbar button. The search bar should appear as in <xref linkend="goscaja-FIG-925"/>
Warning: Could not merge tr translation for msgid:
Select this option to sort the items alphabetically by object type. The items are sorted alphabetically by the description of their <firstterm>MIME type</firstterm>. The MIME type identifies the format of a file, and enables applications to read the file. For example, an email application can use the <literal>image/png</literal> MIME type to detect that a PNG file is attached to an email.
Warning: Could not merge tr translation for msgid:
<imageobject> <imagedata fileref="figures/caja_emblem.png" format="PNG"/> </imageobject> <textobject> <phrase>File icon with Important emblem.</phrase> </textobject>
Warning: Could not merge tr translation for msgid:
<guilabel>Show only folders</guilabel>
Warning: Could not merge tr translation for msgid:
If you prefer, you can assign the double-click action to <firstterm>roll up</firstterm> the window: see <xref linkend="prefs-windows"/>.
Warning: Could not merge tr translation for msgid:
<primary>bottom edge panel</primary> <secondary>default contents</secondary>
Warning: Could not merge tr translation for msgid:
The easiest way to report bugs is by using <_:application-1/>, MATE's built-in bug reporting tool. This will launch automatically in the event that an application crashes. The details MATE developers need are automatically collected, but you can further help by giving information about what you were doing when the crash took place.
Error: Could not merge translations:
'NoneType' object has no attribute 'node'
make[1]: *** [Makefile:569: tr/tr.stamp] Fehler 1
make[1]: Verzeichnis „/media/Work/Programme-linux/Mate-Desktop/git-version/github-matedesktop/mate-user-guide-dist/mate-user-guide“ wird verlassen
make: *** [Makefile:427: all-recursive] Fehler 1
```

I think those are all xml language related issues which needs to be fixed at transifex.
Not sure how we can handle this?
For the moment i disabled `tr` language for 1.21.1 release.

Btw. i got more build issues with some languages if i build against fedora 29 with newer `itstool`.
Eg. uk, ru, da and posible other languages.
I will file out another report for this issues.

#### Steps to reproduce the behaviour
`tx pull` -a -f`
`./autogen.sh && make && make distcheck`

#### MATE general version
1.21.x

#### Package version
1.21.1

#### Linux Distribution
building on fedora 28

#### Link to downstream report of your Distribution
1.21.x isn't in fedora rawhide, yet.

**Comment by lukefromdc** [comment]
As of now, one commit after the 1.21.1 release the build completes fine here on Debian Unstable. I ak guessing https://github.com/mate-desktop/mate-user-guide/commit/eb150d258bda1ffb81304fed43ab47a5567d427e fixed this?

**Comment by raveit65** [comment]
This is a workaround.
`IGNORE_HELP_LINGUAS = tr`
That means the language `tr` won't be translated.
Revert this commit and building will fail.

**Comment by lukefromdc** [comment]
Someone will have to fix that, I don't know anything about working with languages and translations myself. We have a workaround at the moment-but not for users of that language

**Comment by raveit65** [comment]
@yetist 
there is a merge issues with you language.
```
  GEN    zh_CN/
Warning: Could not merge zh-CN translation for msgid:
<_:indexterm-1/><_:indexterm-2/>The icon setting for a theme determines the appearance of the icons on panels and the desktop background. You can choose an option for the icon setting from the <guilabel>Icons</guilabel> tabbed section in the <guilabel>Customize Theme</guilabel> window.
```
The closing `</guilabel>` is missing at transifex server, but i don't know the right position to add it :)
Can you take a look at it please?

**Comment by yetist** [comment]
@raveit65 

I fixed that on transifex, please run `tx pull -l zh_CN -r MATE.master--mate-user-guide-content -f` to download and test.

**Comment by raveit65** [comment]
@yetist 
Thanks this fixes the last remaining merge issue at master branch.
Let see, how the situation is with 1.22 branch.

**Comment by raveit65** [comment]
fixed for 1.22 with
https://github.com/mate-desktop/mate-user-guide/commit/669582fe28e93e6a908c39dc37b43f2570c59859
https://github.com/mate-desktop/mate-user-guide/commit/848d7ae35bf32603101fe6848fdedf435d07fe5f
